### DISCUSSION TITLE
What is the best model or method to make faces similar but not identical, like siblings (brother/sister)? Also, how can I maintain the same style and posture?

### DISCUSSION THREAD

**Discussion by martinenkoEduard** [lllyasviel/Fooocus#3570] [General]
What is the best model or method to make faces similar but not identical, like siblings (brother/sister)? Also, how can I maintain the same style and posture?

Similar to face-swapping, but not exactly like that. I want to create something similar, like a brother or sister.

**Comment by Yogesh-DevHub** [comment]
https://github.com/lllyasviel/Fooocus/discussions/557#discussioncomment-10470471
